Formula/restack.rb

```diff
@@ -0,0 +1,16 @@
+class Restack < Formula
+  desc "CLI app"
+  homepage "https://github.com/restackio/homebrew-restack"
+  url "https://github.com/restackio/cli-bin/archive/refs/tags/v0.1.0.tar.gz"
+  sha256 "417463c85f93594e158a72e0c08362b079d2166fbd663a168e78423eee4e7edb"
+  license "Apache-2.0"
+  head "https://github.com/restackio/cli-bin.git", branch: "v0.1.0"
+
+  def install
+    bin.install "restack"
+  end
+
+  test do
+    assert_match "Restack CLI v0.1.0", shell_output("#{bin}/restack -v")
```

Comment:
Can you propose a more substantial test than checking the version or calling for help? These tests rarely ever fail even for broken installations.

>We want tests that don’t require any user input and test the basic functionality of the application. For example `foo build-foo input.foo` is a good test and (despite their widespread use) `foo --version` and `foo --help` are bad tests. 

Ref. https://docs.brew.sh/Formula-Cookbook#add-a-test-to-the-formula